### ISSUE TITLE
Unfollow with follow_db.db problem

### ISSUE THREAD

**Issue by Massimosso** [instabot-py/instabot.py#1872]
The `follow_db.db` file only populate to 20 users, after it begins to make unfollow to all the other users that have not been inserted in the `follow_db.db` file, including the users that already followed before using the bot (friends, famous people, ecc..)

I would like it to stop following only the accounts followed by the bot itself.

In add to I can not connect other accounts to the bot because the db file is unique for all accounts connected..

I use an Vps server online.. Any solutions?

Check this or we will delete your issue. (fill in the checkbox with an X like so: [x])
- [X] I have searched for other issues with the same problem. (including #1153)

## Hosting
- [X] Are you hosting the bot on a VPS or other Cloud hosting?
- [ ] If so; have you tried running the code on your local machine?

#### Select one:
- [ ] Bug
- [ ] Feature Request
- [X] Technical Help
- [ ] Other

### Operating System? (include version)
- [X] MAC
- [ ] Windows
- [ ] Linux
- [ ] Android

## Python Version?
- [ ] 2.x
- [X] 3.x

## Description of your issue

```
Include the contents of the error.log here (if applicable). < REPLACE THIS TEXT
```

## Config

```
Include the contents of your config here (REMOVE YOUR USERNAME AND PASSWORD!) (if applicable). < REPLACE THIS TEXT
```

**Comment by dimisdas** [comment]
I agree -- the auto bot should also get new users from recent feed periodically and insert into DB. 

Todo.

**Comment by Massimosso** [comment]
@dimisdas Exactly, the bot just stopped following my instagram friends, so I had to block it. Also I can not even link multiple accounts at the same time

**Comment by dimisdas** [comment]
You can link as many accounts you want on different instances of the bot.

 Either duplicate example.py and change the .db, or use run.py as many times you want.

**Comment by dimisdas** [comment]
Yes

**Comment by Massimosso** [comment]
What should I change in example.py? I can not find the words "follows_db.db" in example.py file

**Comment by Massimosso** [comment]
@dimisdas What should I change in example.py? I can not find the words "follows_db.db" in example.py file

**Comment by JD2719** [comment]
Same problem here. I dont think the follows_db is populating. 

Has anyone had any luck running a different mode?

**Comment by Massimosso** [comment]
It’s sure not populating. One hour ago the bot unfollows my brother 😂

**Comment by worldsdream** [comment]
Yes, it should only unfollow what the bot friended. It’s not doing that. Waiting for it.

**Comment by Massimosso** [comment]
Any news? @dimisdas

**Comment by Massimosso** [comment]
> @dimisdas What should I change in example.py? I can not find the words "follows_db.db" in example.py file

I can not create multiple .db files without run.py ! Any solutions for connect multiple account with more .db files?

**Comment by dimisdas** [comment]
Guys, it's been implemented on https://github.com/instabot-py/instabot.py/tree/session_persist

Quoting the changelog:

> Changelog
> 
> Changed version from 1.2.0 to 1.2.1
> run.py updated and will now ask comma delimited tags to save on config.ini
> run.py updated to support multiple persisting sessions
> instabot class updated to accept session_file (str) option, where it will save the login cookies and reuse them (persistent sessions)
> **populate_from_feed() function now called at the same intervals as auto_mod_unfollow, to keep the database populated with users other than the people the bot has followed.**

**Comment by Massimosso** [comment]
@dimisdas nothing.. The bot continue to unfollow my friends from Instagram. It should stop following only users who followed him.

In add to the follows_db.db is still unique for all accounts: if I do it popular with the first account, when I connect the second account, this does not make the population, but is based only on the .db file of the first account. 

It is identical to before 😥

**Comment by dimisdas** [comment]
The bot has no way of knowing who "followed" him. It unfollows people who match some criteria (don't follow you back, are celebrity accounts, spam, etc).

You can set another .db name in example.py for each user like so:

```
bot = InstaBot(
    login="username",
    password="password",
    database_name="my_other_db.db"
    like_per_day=1000,
    comments_per_day=0,

```

You can also use run.py that does that automaticaly for you.

**Comment by Massimosso** [comment]
@dimisdas ... It’s no possible change this criteria? 
Two days ago the bot stops to follow my brother only because he don’t follow me 😥

**Comment by dimisdas** [comment]
You should whitelist him ... or ask him to follow you back, perhaps?

Or whatever. I'm not getting involved in any more family drama than I have on my plate right now.
 🙉 🙈 🙊
